src/protocols/TRXLooper.cpp

```diff
@@ -339,8 +342,9 @@ OpStatus TRXLooper::RxSetup()
 #endif
 
     // wait for Rx thread to be ready
+    std::unique_lock lck{ mTx.mutex };
     while (mRx.stage.load(std::memory_order_relaxed) < Stream::ReadyStage::WorkerReady)
-        ;
+        mTx.cv.wait(lck);
```

Comment:
Again, poorly tested. Wrong conditional variable, causes lock up. Fixed in 'develop'